Superbuild/External-ITK.cmake

```diff
@@ -2,7 +2,7 @@
 # Get and build itk
 
 if(NOT ITK_TAG)
-  set(ITK_TAG "v5.0rc01")
+  set(ITK_TAG "v5.0.0")
```

Comment:
#103 also updates this, and fixes some other things too.

Reply:
I know, but #103 is not yet merged. Can you rebase and merge #103 first?